pkg/steps/pod.go

```diff
@@ -191,6 +193,15 @@ func PodStep(name string, config PodStepConfiguration, resources api.ResourceCon
 	}
 }
 
+func getEnv(s *podStep) []coreapi.EnvVar {
```

Comment:
We already have the spec -- call the `decorate.KubeEnv(downwardapi.EnvForSpec())` on `podStep.jobSpec` and we don't need this method

Reply:
Done.